site_scons/state.py

```diff
@@ -75,55 +75,60 @@ def _getBinPath(binName, msg=None):
         log.debug("Found %s here %s" % (binName, binFullPath))
         return binFullPath
 
+
 def _getBinPathFromBinList(binList, msg=None):
     binFullPath = None
-    i=0
-    if msg == None:
+    i = 0
+    if msg is None:
         msg = "Looking for %s" % binList
     log.info(msg)
     while i < len(binList) and not binFullPath:
         binName = binList[i]
         binFullPath = SCons.Util.WhereIs(binName)
-        i=i+1
+        i = i + 1
     if not binFullPath:
-            raise SCons.Errors.StopError('Could not locate at least one binary in : %s' % binList)
+        raise SCons.Errors.StopError('Could not locate at least one binary in : %s' % binList)
     else:
         return binFullPath
 
+
 def _findPrefixFromBin(key, binName):
     """ returns install prefix for  a dependency named 'product'
     - if the dependency binary is PREFIX/bin/binName then PREFIX is used
     """
-    prefix = _findPrefixFromPath(key,  _getBinPath(binName))
+    prefix = _findPrefixFromPath(key, _getBinPath(binName))
     return prefix
 
+
 def _findPrefixFromPath(key, binFullPath):
-    if not binFullPath :
+    if not binFullPath:
         log.fail("_findPrefixFromPath : empty path specified for key %s" % key)
     (binpath, binname) = os.path.split(binFullPath)
     (basepath, bin) = os.path.split(binpath)
     if bin.lower() == "bin":
         prefix = basepath
 
     if not prefix:
-        log.fail("Could not locate install prefix for product containing next binary : " % binFullPath )
+        log.fail("Could not locate install prefix for product containing next binary : " % binFullPath)
```

Comment:
`%s` is missing in a format string, and it really should read:

    log.fail("Could not locate install prefix for product containing next binary : %s", binFullPath)